samples/matter/lock/Kconfig.sysbuild

```diff
@@ -8,6 +8,10 @@
 config NRF_DEFAULT_IPC_RADIO
 	default y
 
+# Use HCI IPC as radio core image for nRF54H20 + nRF7002EK
+config NRF_DEFAULT_BLUETOOTH
+	default y if SOC_SERIES_NRF54HX && WIFI_NRF700X
```

Comment:
radio IPC does not work for nRF54H20 with WiFi?

Reply:
Something doesn't work here with WiFi, for now, we need to use hci_ipc and in the next step, we will switch to radio_ipc. Are you OK with that?

Reply:
Currently, there is an additional dts overlay file as a workaround for radio IPC that works only if IEEE802154 is enabled. For the WiFi variant, we don't need it so for now I switched to hci_ipc. In the future, I'm going to use radio IPC for this purpose as well, once it is ready.